src/ORM/TableRegistry.php

```diff
@@ -184,7 +184,7 @@ public static function get($alias, array $options = [])
         }
 
         if (isset(static::$_config[$alias])) {
-            $options += static::$_config[$alias];
+            $options = static::$_config[$alias] + $options;
```

Comment:
Sorry misread, merge is done with `static::$_config[$alias]` not `static::$_options[$alias]`.